src/masonite/commands/presets/Remove.py

```diff
@@ -0,0 +1,51 @@
+"""Remove Preset"""
+import os
+import shutil
+from ..presets import Preset
+
+
+class Remove(Preset):
+    """Removes any defined Preset"""
+
+    def install(self):
+        """Install the preset"""
+        self.update_packages()
```

Comment:
Are these three needed here ? Especially since we are removing stuff.

Reply:
I’m not sure I understand? Those functions all do something. So they should be there and they are not no-ops